guides/micronaut-scope-types/java/src/test/java/example/micronaut/PrototypeScopeTest.java

```diff
@@ -0,0 +1,53 @@
+//tag::imports[]
+package example.micronaut;
+
+import io.micronaut.core.type.Argument;
+import io.micronaut.http.HttpRequest;
+import io.micronaut.http.HttpResponse;
+import io.micronaut.http.HttpStatus;
+import io.micronaut.http.client.BlockingHttpClient;
+import io.micronaut.http.client.HttpClient;
+import io.micronaut.http.client.annotation.Client;
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
+import jakarta.inject.Inject;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@MicronautTest // <1>
+class PrototypeScopeTest {
+    @Inject
+    @Client("/")
+    HttpClient httpClient; // <2>
+
+//end::imports[]
+/*
+//tag::testheader[]
+        @Test
+        void prototypeScopeIndicatesThatANewInstanceOfTheBeanIsCreatedEachTimeItIsInjected() {
+            String path = "/";
+//end::testheader[]
+*/    
```

Comment:
This is rendered really ugly:

<img width="1178" alt="image" src="https://user-images.githubusercontent.com/153880/191548788-d0677ccb-e9fe-4019-8dc9-c19c305208f0.png">

Perhaps indent back 1 level the commented-out code?

Reply:
resolved via https://github.com/micronaut-projects/micronaut-guides/pull/1085/commits/0f1be2e9d2d1435884a8f2d5a952db5e696d4610